token-lending/program/src/state/lending_market.rs

```diff
@@ -116,13 +120,21 @@ impl Pack for LendingMarket {
             input,
             1,
             1,
+            1,
             PUBKEY_BYTES,
             32,
             PUBKEY_BYTES,
             PUBKEY_BYTES,
             128
         ];
 
+        if *account_type != AccountType::LendingMarket.to_le_bytes()
+            && *account_type != AccountType::Unitialized.to_le_bytes()
```

Comment:
This is something I had to add that is kind of annoying.  Loading an account with empty data is a valid thing to do since in instructions like init_lending_market, we first deserialize all of the account data to make sure that it's empty. A bunch of the tests were breaking here since the account type was 0. Given that this check is just for differentiating between different account types, it seems somewhat okay to me though.